go/libraries/doltcore/schema/typeinfo/inlineblob.go

```diff
@@ -43,6 +43,20 @@ func (ti *inlineBlobType) ConvertNomsValueToValue(v types.Value) (interface{}, e
 	return nil, fmt.Errorf(`"%v" cannot convert NomsKind "%v" to a value`, ti.String(), v.Kind())
 }
 
+// ReadFrom reads a go value from a noms types.CodecReader directly
+func (ti *inlineBlobType) ReadFrom(_ *types.NomsBinFormat, reader types.CodecReader) (interface{}, error) {
+	k := reader.ReadKind()
```

Comment:
Panic here for now as I've changed all of this logic, and I don't wanna accidentally forget to make the change on my PR.

Reply:
Wait no this is wrong, ignore that don't panic here. I'll leave another suggestion for what it should be.